enclave/src/rpc/worker_api_direct.rs

```diff
@@ -16,6 +16,19 @@
 */
 
 pub extern crate alloc;
+
```

Comment:
The `use` changes here are essentially just a grouping together and letting the formatter order them